lib/WP_Auth0_LoginManager.php

```diff
@@ -587,6 +576,68 @@ public function end_session() {
 		}
 	}
 
+	/**
+	 * Get and filter the scope used for access and ID tokens
+	 *
+	 * @param string $context - how are the scopes being used?
+	 *
+	 * @return string
+	 */
+	public static function get_userinfo_scope( $context = '' ) {
```

Comment:
Moved default scopes to here so they can be filtered

Reply:
Also `$context` is only for that filter. I pass in a context when I request those scopes so the filter function can decide to do something different based on what's being asked.

Reply:
how does this method work? I saw you call it passing `sso` and `lock`. The method should have a single known behavior. What's the input/output?

Reply:
On a basic install with no customizations, 'openid email name nickname picture' will always be returned. If someone wants to modify the scopes that are being requested, they can use the `auth0_auth_token_scope` filter to add/remove. The `$context` is there to provide ... context for where those scopes are used. This is a pattern used in i18n in WordPress where a word might be different depending on context.  

Adjusted this to use an array for filtering ... better data structure for what's being done here.